shell/platform/android/io/flutter/plugin/editing/InputConnectionAdaptor.java

```diff
@@ -75,6 +75,11 @@ public boolean equals(Object o) {
           && composingEnd == value.composingEnd
           && text.equals(value.text);
     }
+
+    @Override
+    public int hashCode() {
```

Comment:
The pattern followed elsewhere (I looked at AccessibilityViewEmbedder.java) seems to combine the hashcode with a prime.  This one looks fine to me but it may be a good idea to isolate that into a helper so this doesn't have to be reinvented.

Reply:
I'll follow the same style as AccessibilityViewEmbedder.java for now to unblock the G3 roll, but yeah, we should find a place to add prime hashing logic to share across the android embedder.